welcome.html

```diff
@@ -16,8 +16,8 @@
           <P class="version-tag">v1.0.0</P>
```

Comment:
This is about the part above.
The class name is `web-name-box`, but if you do not want to use it in the CSS file, leave only the `div `tag and delete the class name `web-name-box`.

Reply:
I deleted the class name 'web-name-box' 
Thank you